shared/testutil/helpers.go

```diff
@@ -20,11 +21,16 @@ var lock sync.Mutex
 
 // Caches
 var deposits []*ethpb.Deposit
+var depositDataRoots [][32]byte
 var privKeys []*bls.SecretKey
 var trie *trieutil.MerkleTrie
 
-// SetupInitialDeposits prepares the entered amount of deposits
-// and secret keys.
+// SetupInitialDeposits prepares the entered amount of deposits,
+// deposit data roots, and secret keys.
+// The deposits are configured such that for deposit n the validator
+// account is key n and the withdrawal account is key n+1.  As such,
+// if all secret keys for n validators are required then numDeposits
+// should be n+1
```

Comment:
shouldnt this be 2n ?

Reply:
No; key n is the validator and n+1 the withdrawal key for *each* n.  So if you asked for 3 deposits, for example:

  - key 0 would be validator for deposit 0
  - key 1 would be withdrawal for deposit 0 and validator for deposit 1
  - key 2 would be withdrawal for deposit 1 and validator for deposit 2
  - key 3 (unreturned) would be validator for deposit 2

The point of the change is to make the validator and withdrawal keys different for each deposit rather than unique across all deposits.